client/src/_reducers/auth.js

```diff
@@ -0,0 +1,20 @@
+import dayjs from 'dayjs'
+import { ADD_AUTH, REMOVE_AUTH } from '_actions/auth'
+
+const initialAuthState = {
+  token: localStorage.getItem('pie-token'),
+  expiration: localStorage.getItem('pie-expiration') || dayjs().toDate()
+}
+
+export default function auth(state = initialAuthState, action) {
+  switch (action.type) {
+    case ADD_AUTH:
```

Comment:
follow up with my comment above: this case would be where you return the Object below. and the REMOVE_AUTH case would update the `token` to `null. 

related question for my own clarity -- expiration date is created upon adding the token, and then when removed, we keep it, or is it also removed? (to redux state, not localStorage)